client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/task-helper.ts

```diff
@@ -91,6 +94,42 @@ export function getEnhancedTasks(
 						},
 					};
 					break;
+				case 'videopress_setup':
+					taskData = {
+						title: translate( 'Set up your Video Site' ),
```

Comment:
```suggestion
						title: translate( 'Set up your Video site' ),
```
Not entirely sure about this, but to match the casing of the initial launch page, `site` is lowercased there as well.